hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/DBCheckpointServlet.java

```diff
@@ -195,7 +195,9 @@ private void generateSnapshotCheckpoint(HttpServletRequest request,
               .filter(s -> s.endsWith(ROCKSDB_SST_SUFFIX))
               .distinct()
               .collect(Collectors.toList()));
-      LOG.info("Received excluding SST {}", receivedSstList);
+      if (sstParam.length % 10 == 0) {
```

Comment:
what I meant by 5 or 10 was that we can print only 5  filenames in the LOG.info output and say there are few more

Eg:

`INFO Received excluding SST : { 1.sst 2.sst 3.sst 4.sst 5.sst } . The total excluded sst files count is 125.`

Under debug we can keep as is

``` java
if (LOG.isDebugEnabled()) {
    LOG.debug("Received excluding SST {}", receivedSstList);
}
```